projects/nutanix-cloud-native/cluster-api-provider-nutanix/build/create_manifests.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -x
+set -o errexit
+set -o pipefail
+
+REPO="$1"
+OUTPUT_DIR="$2"
+ARTIFACTS_PATH="$3"
+TAG="$4"
+IMAGE_REPO="$5"
+IMAGE_TAG="$6"
+GOLANG_VERSION="$7"
+
+MAKE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${MAKE_ROOT}/../../../build/lib/common.sh"
+
+build::common::use_go_version $GOLANG_VERSION
+
+cd $REPO
+
+yq eval -i -P ".spec.template.spec.containers[0].args += [\"--namespace=eksa-system\"]" config/manager/manager.yaml
```

Comment:
Why do we need to pass the namespace to the container?

Also, I think we will need to set the image repo and tag for the CAPX deployment. Here's a couple examples
https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/tinkerbell/cluster-api-provider-tinkerbell/build/create_manifests.sh#L33-L36
https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/cluster-api-provider-vsphere/build/create_manifests.sh#L41